flask_mongorest/resources.py

```diff
@@ -83,7 +96,7 @@ def raw_data(self):
                     raise ValidationError({'error': "Chunked Transfer-Encoding is not supported."})
 
                 try:
-                    self._raw_data = json.loads(request.data)
+                    self._raw_data = json.loads(request.data, parse_constant=self._enforce_strict_json)
```

Comment:
I would have assumed the basic issue is that we have a `json.dumps` without using allow_nan=False` (e.g. like in views.py). Don't we need to handle it there?